packages/react/src/components/Masthead/MastheadL1.js

```diff
@@ -20,21 +21,20 @@ const { prefix } = settings;
  * @typedef {object} navigation Object containing navigation elements
  * @returns {*} Masthead component
  */
-const MastheadL1 = () => {
+const MastheadL1 = ({ isShort, title, eyebrowLink }) => {
   const className = cx({
     [`${prefix}--masthead__l1`]: true,
+    [`${prefix}--masthead__l1--short`]: isShort,
   });
 
   return (
     <div className={className}>
       <div className={`${prefix}--masthead__l1-name`}>
         <span className={`${prefix}--masthead__l1-name-eyebrow`}>
           <ArrowLeft16 />
-          <a href="#">Eyebrow</a>
-        </span>
-        <span className={`${prefix}--masthead__l1-name-title`}>
-          Stock Charts
+          <a href={eyebrowLink}>Eyebrow</a>
         </span>
+        <span className={`${prefix}--masthead__l1-name-title`}>{title}</span>
       </div>
       <HeaderNavigation className={`${prefix}--masthead__l1-nav`} aria-label="">
         <HeaderMenuItem href="#">Link 1</HeaderMenuItem>
```

Comment:
@kennylam @emerson-pereira I was originally going to ask about getting the navigation items as props, but I think is out of the scope of this PR/issue and also it's still not concrete yet, right?

Reply:
@jeffchew @kennylam I rolled back props out of this PR and left it strictly for short L1 fix. I'll create a new PR for props after this.